client/src/modules/authorization/model/actions.js

```diff
@@ -26,7 +26,10 @@ const loginSuccess = data => ({
 
 export const logoutCurrentUser = () => dispatch =>
   postRequest('/auth/logout')
-    .then(() => dispatch(logoutSuccess()))
+    .then(() => {
+      dispatch(logoutSuccess());
+      window.location.reload();
```

Comment:
on what page we will be after logging out?

Reply:
on login page